docs(README.md): Update documentation with latest scope (#326)
SAP/ui5-linter: README.md

UI5 linter is a static code analysis tool for UI5 projects.
It checks JavaScript, TypeScript, XML, JSON, and other files in your project and reports findings.

## Features
UI5 linter scans your UI5 project ands detects code which won't work when running your project with [UI5 2.x](https://community.sap.com/t5/open-source-blogs/introducing-openui5-2-x/ba-p/13580633). 

> * Detect use of deprecated UI5 libraries 

Reviewer: ```suggestion
* Usage of deprecated UI5 libraries 
```

> * Detect use of deprecated APIs provided by the UI5 framework 

Reviewer: ```suggestion
* Usage of deprecated UI5 framework APIs
```

* Usage of global variables 
* CSP best practices 
* Component best practices 

> * Recommend best practices in manifest.json 

Reviewer: ```suggestion
* Best practices for manifest.json 
```

* Support for JavaScript and TypeScript projects 

> [!NOTE]

> > UI5 linter already provides many detections, it is not yet covering all aspects and not all best practices for UI5 2.x. The intention of UI5 linter is to detect as many as possible findings not compliant with UI5 2.x. However it’s still necessary to test your UI5 project with UI5 2.x as soon as it is made available. The UI5 team plans more versions of UI5 linter over the next months, which can reveal more findings. 

Reviewer: ```suggestion
> Wile UI5 linter already provides many detection features, it is not yet covering all aspects and best practices for UI5 2.x. The intention of UI5 linter is to detect as many issues as possible that a project running with UI5 2.x might be facing. However, you'll still need to test your UI5 project with UI5 2.x as soon as it is made available. To reveal additional issues, the UI5 team plans to release more versions of UI5 linter over the next months. 
```
